wiki/Graveyard/de.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+no_native_review: true
+tags:
+  - dead
+---
+
+# Graveyard
+
+Der **Graveyard** (auf Deutsch: Friedhof) ist eine Unterform von [Beatmaps](/wiki/Beatmap), welche nicht mehr von ihren Erstellern (Mappern) beachtet werden. 
```

Comment:
I wouldn't translate _subforum_ as _Unterform_ (maybe this was an inadvertent mistake anyway?). The literal translation sounds alright to me (_Unterforum_), but I think _Unterkategorie_ would be better.

Also, it's probably clear what _Ersteller_ means, so the _Mappern_ in brackets feels a bit redundant.